Lib/random.py

```diff
@@ -242,6 +242,8 @@ def _randbelow(self, n, int=int, maxsize=1<<BPF, type=type,
                 "enough bits to choose from a population range this large.\n"
                 "To remove the range limitation, add a getrandbits() method.")
             return int(random() * n)
+        if n == 0:
```

Comment:
PEP 8 suggests `not n`.

Reply:
The condition ``n == 0`` mirrors the docstring,  ``Raises ValueError if n==0."``

If running code with the ``-W error`` option, when the sequence is very large this will not reach the ``if n == 0`` condition. Perhaps it would be better to place the condition earlier in the function?